llama-index-integrations/embeddings/llama-index-embeddings-ipex-llm/llama_index/embeddings/ipex_llm/utils.py

```diff
@@ -0,0 +1,63 @@
+from typing import Optional, List
+from pathlib import Path
+
+DEFAULT_HUGGINGFACE_EMBEDDING_MODEL = "BAAI/bge-small-en-v1.5"
+DEFAULT_INSTRUCT_MODEL = "hkunlp/instructor-base"
+
+# Originally pulled from:
+# https://github.com/langchain-ai/langchain/blob/v0.0.257/libs/langchain/langchain/embeddings/huggingface.py#L10
+DEFAULT_EMBED_INSTRUCTION = "Represent the document for retrieval: "
+DEFAULT_QUERY_INSTRUCTION = (
+    "Represent the question for retrieving supporting documents: "
+)
+DEFAULT_QUERY_BGE_INSTRUCTION_EN = (
+    "Represent this question for searching relevant passages: "
+)
+DEFAULT_QUERY_BGE_INSTRUCTION_ZH = "为这个句子生成表示以用于检索相关文章："
```

Comment:
i'm not sure if adding chinese in code is good. I know this is a default instruction. Can we read it from somewhere?

Reply:
or maybe remove the chinese version support for now?